tests/unit/bokeh/plotting/test_figure.py

```diff
@@ -63,10 +63,18 @@ def test_width_height(self) -> None:
         assert p.plot_width == 100
         assert p.plot_height == 120
 
+        p = bpf.figure(width=100, height=120)
+        assert p.width == 100
+        assert p.height == 120
+
         p = bpf.figure(plot_width=100, plot_height=120)
         assert p.plot_width == 100
         assert p.plot_height == 120
 
+        p = bpf.figure(plot_width=100, plot_height=120)
+        assert p.width == 100
+        assert p.height == 120
```

Comment:
Instead of duplicating those tests, you should check if `p.plot_with` and `p.width` are the same under both ways of initializing `figure()`, i.e. `assert p.width == p.plot_width == 100`, as not only it's shorter, but also states the intent of this test more clearly.